a.core_concepts.md

```diff
@@ -189,6 +200,18 @@ wget -O- 10.1.1.131:80
 exit
 ```
 
+Alternatively you can also try a more advanced option:
+
+```bash
+# Get IP of the nginx pod
+NGINX_IP=$(kubectl get pod nginx -o jsonpath='{.status.podIP}');
```

Comment:
doesn't work for me if you include the colon (`;`) at the end. Works if you take it out, give it a shot!

Reply:
sorry !, done